smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/serde/KeyDeserializerConfigurationTest.java

```diff
@@ -12,16 +12,6 @@
 import javax.enterprise.inject.AmbiguousResolutionException;
 import javax.enterprise.inject.UnsatisfiedResolutionException;
 
-import io.smallrye.reactive.messaging.kafka.CountKafkaCdiEvents;
```

Comment:
Ashy so many import reorder?

Reply:
I'll double-check but I think it comes from a previously merged PR.